Protocol.md

```diff
@@ -133,15 +160,15 @@ If Metadata Length is greater than this value, the entire frame MUST be ignored.
      0                   1                   2                   3
      0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-    |R|                       Metadata Length                       |
+    |              Metadata Length                  |
     +-+-------------------------------------------------------------+
```

Comment:
I should have been clearer. What I was saying is that maybe we can trim this diagram to 24 bits. If we are consistently using 32 bits for all diagrams then `Framing Format` diagram is an outlier.